main.js

```diff
@@ -1,7 +1,9 @@
 // Modules to control application life and create native browser window
 import {app, BrowserWindow} from 'electron'
-import {checkWhichHost, curseForge} from './src/parsePage'
+import {checkWhichHost, parseAddonDetails_curseforge} from './src/parsePage'
```

Comment:
are we doing camelcase or something mixed for this project?

Reply:
Camelcase!

Reply:
Cool I'll refactor this when I'm converting the project to hexagonal architecture.